bots/images/scripts/lib/fedora.install

```diff
@@ -91,6 +92,9 @@ else
     echo '====== skipping rpmlint check, not installed ====='
 fi
 EOF
+    chmod +x /tmp/rpmlint
+    su builder -c "/usr/bin/mock --offline --copyin /tmp/rpmlint /var/tmp/rpmlint"
+    su builder -c "/usr/bin/mock --offline --shell /var/tmp/rpmlint"
```

Comment:
Can you please split this out into a separate commit (or even PR) with a more detailled explanation?